rebase.go

```diff
@@ -20,6 +20,14 @@ type RebaseOptions struct {
 }
 
 func (c *Client) Rebase(ctx context.Context, opts RebaseOptions) error {
+	info, err := c.docker.Info(ctx)
+	if err != nil {
+		return errors.Wrap(err, "getting docker host OS")
+	}
+	if info.OSType == "windows" {
```

Comment:
This seems to be validating the docker OS, rather than the OS of the image we are rebasing. Should we also handle the case of someone trying to rebase a Windows container on another OS? That is to say, I run `pack rebase <some-windows-container>` on my `mac`?

Reply:
The two are one and the same, as you can only run Windows containers on a Windows docker host (and the other direction is true too: a Windows docker host can only run Windows containers, and not Linux, etc). If you happen to run commands on your Mac that are intended for Windows containers/images, you'd need to set the `DOCKER_HOST` env var to point to a Windows docker host (in fact, this is what we do on a daily basis).

Reply:
That said, I think this should actually not be an `ExperimentError`. That was likely a copy/paste artifact.

Reply:
Also, I should clarify the `info.OSType` is the docker __host__ OS, not necessarily the OS running the docker client. If you have `DOCKER_HOST` set to point to a Windows machine configured to run Windows images, the value of `OSType` will be `windows`, even if you're executing `pack` from a Mac.